app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Assessment only route to QTS"
-heading: "Assessment only route to QTS for unqualified teachers"
+title: "Train to teach if you’ve worked as an unqualified teacher"
```

Comment:
I think we need to keep `Assessment only` - and probably in both the title and h1 - as this is a big organic search term for us and is a named route that people look for and understand. And out of the search terms containing this, 'Assessment only route to QTS' is the one that brings most clicks.

Should we change the nav title instead?

Our unqualified teacher traffic tends to be around salary - and we link to this assessment only page from the pay and benefits page

But happy to discuss if you have a strong SEO/navigation reason to change this!

Reply:
'Train to teach by Assessment only' or have it in brackets at the end of this title?